src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
-import React from 'react';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
```suggestion
export const MoviesList: React.FC<Props> = ({ movies  }) => (
```
Movies is required prop